SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -18,6 +18,9 @@ struct FConnectionConfig
 		, EnableProtocolLoggingAtStartup(false)
 		, LinkProtocol(WORKER_NETWORK_CONNECTION_TYPE_MODULAR_UDP)
 		, TcpMultiplexLevel(2) // This is a "finger-in-the-air" number.
+		, TcpNoDelay(0)
+		, UdpUpstreamIntervalMS(10)
```

Comment:
I'd rather put `0` and not `10` here, to explicitly state we init it with default values and `PreConnectInit` must be called to read proper values from the Settings.

Reply:
Fair enough